crates/file-explorer/src/fs.rs

```diff
@@ -115,6 +117,7 @@ pub struct Directory {
 }
 
 /// An entry in a directory
+#[allow(dead_code)]
```

Comment:
```suggestion
```

Reply:
I don't know why there is dead code in the crate file-explorer but the warnings are present.